### ISSUE TITLE
Edit user page

### ISSUE THREAD

**Issue by ahansson89** [AnomalyInnovations/serverless-stack-com#157]
I would love to see an example on how to edit userdata, such as email, address, phone number as well as a custom field, like favorite color.

The reason that I would like this example is to do more comprehensive stuff with the Cognito user table.

**Comment by jayair** [comment]
@ahansson89 Yeah I can see that as a good topic for an "extra credit" chapter.

For the time being, if you want to use the attributes in Cognito the **Use case 8** in this page can help you https://github.com/aws/amazon-cognito-identity-js.
